src/xcode/ENA/ENA/Source/Scenes/ContactDiary/AddAndEditEntry/TextFieldManager.swift

```diff
@@ -0,0 +1,75 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+final class TextFieldsManager {
+
+	// MARK: - Init
+
+	/** Manager to handle multiple TextFiled, each gets an associated WritableKeyPath */
+	init() {
+		self.textFieldsWithKeyPaths = []
+	}
+
+	// MARK: - Internal
+
+	// helper to get all textfiled without attached KeyPath
+	var textFiledsOnly: [UITextField] {
+		textFieldsWithKeyPaths.map { textFiled, _ -> UITextField in
+			return textFiled
+		}
+	}
+
+	// get the associated WritableKeyPath for a textfield - if known
+	func keyPath(for searchTextField: UITextField) -> WritableKeyPath<DiaryAddAndEditEntryModel, String>? {
+		textFieldsWithKeyPaths.first { textField, _ -> Bool in
+			searchTextField == textField
+		}?.1
```

Comment:
Not sure, could this `1` be named somehow?

Reply:
done